integration-tests/integration/alerts/createAlert.spec.js

```diff
@@ -0,0 +1,60 @@
+const offenderBasicDetails = require('../../mockApis/responses/offenderBasicDetails.json')
+const offenderFullDetails = require('../../mockApis/responses/offenderFullDetails.json')
+const CreateAlertPage = require('../../pages/alerts/createAlertPage')
+const PrisonerAlertsPage = require('../../pages/prisonerProfile/prisonerAlertsPage')
+
+context('A user can add an appointment', () => {
+  before(() => {
+    cy.clearCookies()
+    cy.task('reset')
+    cy.task('stubLogin', { username: 'ITAG_USER', caseload: 'MDI' })
+    cy.login()
+  })
+  beforeEach(() => {
+    Cypress.Cookies.preserveOnce('hmpps-session-dev')
+    cy.task('resetAndStubTokenVerification')
+    const offenderNo = 'A12345'
+    cy.task('stubOffenderBasicDetails', offenderBasicDetails)
+    cy.task('stubAlertTypes')
+    cy.task('stubCreateAlert')
+
+    cy.visit(`/offenders/${offenderNo}/create-alert`)
+  })
+
+  it('A user can successfully add an alert', () => {
+    cy.task('stubClientCredentialsRequest')
+    cy.task('stubPrisonerProfileHeaderData', {
+      offenderBasicDetails,
+      offenderFullDetails,
+      iepSummary: {},
+      caseNoteSummary: {},
+    })
+    cy.task('stubAlertsForBooking', [])
+    const createAlertPage = CreateAlertPage.verifyOnPage()
+    const form = createAlertPage.form()
+    form.alertType().select('F1')
+    // eslint-disable-next-line cypress/no-unnecessary-waiting
+    cy.wait(50)
```

Comment:
Need to do this to give time for the updated sub type select dropdown to reattach itself to the DOM.

Reply:
Yeah, we need to find a solution for this as it's the same issue with case notes.